ui/components/multichain/toast/toast.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { ThemeType } from '../../../../shared/constants/preferences';
+import { BannerBase, Box, ButtonLink, Text } from '../../component-library';
+import { Display } from '../../../helpers/constants/design-system';
+
+export const ToastContainer = ({
+  children,
+}: {
+  children: React.ReactNode | string;
+}) => <Box className="toasts-container">{children}</Box>;
+
+export const Toast = ({
+  startAdornment,
+  text,
+  actionText,
+  onActionClick,
+  onClose,
+}: {
+  startAdornment: React.ReactNode | React.ReactNode[];
+  text: string;
+  actionText: string;
+  onActionClick: () => void;
+  onClose: () => void;
+}) => {
+  const { theme } = document.documentElement.dataset;
+
+  return (
+    <BannerBase
```

Comment:
We need to pass a className prop here

Reply:
We could but we aren't using it anywhere.